tools/scrape/github.py

```diff
@@ -53,6 +53,19 @@ def __str__(self):
         return repr(self.headers)
 
 
+def get_all_info_from_url(github_url):
+    info = re.compile('^https://github.com/([^/]+)/([^/]+)')
```

Comment:
nit: `re.compile` is meant to be run once, as an optimization. since you have it in the function body itself here it doesn't offer any optimization; might as well directly call `re.findall` in the code below. also I'd avoid re-using variable names like this, especially when their types change (from regex to regex match)

Reply:
I moved the `re.compile` out.